central/policy/store/boltdb/store.go

```diff
@@ -0,0 +1,150 @@
+package boltdb
+
+import (
+	"context"
+
+	policyUtils "github.com/stackrox/rox/central/policy/utils"
+	"github.com/stackrox/rox/central/role/resources"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/bolthelper"
+	"github.com/stackrox/rox/pkg/defaults/policies"
+	"github.com/stackrox/rox/pkg/errorhelpers"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/set"
+	"github.com/stackrox/rox/pkg/utils"
+	bolt "go.etcd.io/bbolt"
+)
+
+var (
+	policyBucket = []byte("policies")
+	// As of 66.0, any default policies that were added from 65.0 onwards cannot be deleted.
+	// Any policies added prior to 65.0 can be deleted because the criteria fields are not locked.
+	// Locked policy criteria guarantees that the criteria remains unchanged as is as it was shipped.
+	removedDefaultPolicyBucket = []byte("removed_default_policies")
+
+	policyCtx = sac.WithGlobalAccessScopeChecker(context.Background(),
```

Comment:
policyCtx = context.Background()

We don't need sac inside this store because there is no evaluation for it within bolt

Reply:
ok didn't change this bit (just moved the location of the file) but changing to context.Background